frame/evm/src/lib.rs

```diff
@@ -15,7 +15,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! EVM execution module for Substrate
+//! # EVM Module
+//!
+//! The EVM module allows unmodified EVM code to be executed in a Substrate-based blockchain.
+//! - [`evm::Trait`](https://docs.rs/pallet-evm/2.0.0/pallet_evm/trait.Trait.html)
+//!
+//! ## EVM Engine
+//!
+//! The EVM module uses [`SputnikVM`](https://github.com/rust-blockchain/evm) as the underlying EVM engine. The engine is overhauled so that it's [`modular`](https://github.com/corepaper/evm). In the future, we will want to allow users to swap out components like gasometer, and inject their own customized ones.
```

Comment:
Probably not "in the future" any more. The "native" runner vs "stack" runner is already doing this. (A small note is that we're changing the slightly confusing name of "native runner" to something else, but allow me to refer to it as this still right now.)

My suggestion is that we just remove this line.
```suggestion
//! The EVM module uses [`SputnikVM`](https://github.com/rust-blockchain/evm) as the underlying EVM engine. The engine is overhauled so that it's [`modular`](https://github.com/corepaper/evm).
```